grml_tips

```diff
@@ -3258,3 +3258,29 @@ especially arp spoofing. Do not use in hostile networks.
 
 Tags: aoe, blockdevice, export, client
 -- 
+Check notebook's battery status:
+
+% acpi -b
+
+or
+
+% cat /sys/class/power_supply/BAT0/capacity
+
+Tags: client, notebook
+-- 
+Change notebook's screen brightness:
+
```

Comment:
cat or echo?

Reply:
`cat`, to View the battery charge level one need to read  the capacity file:
```
❯ cat /sys/class/power_supply/BAT0/capacity
90
```

There are other ways to do this, but I believe these are the simplest and work OOTB in Grml.

Reply:
You are right. We need echo here...

Reply:
Thx. I updated the PR.